aten/src/ATen/quantized/Quantizer.cpp

```diff
@@ -44,7 +44,7 @@ inline QTensor new_qtensor(
       allocator,
       /*resizable=*/true);
   auto tensor = detail::make_tensor<QTensorImpl>(
-      storage_impl, at::CPUTensorId(), is_variable, quantizer);
+      storage_impl, at::AffineCPUTensorId(), is_variable, quantizer);
```

Comment:
if we have different type_ids for QTensors how do we map correctly from the quantizer to the Id?  Maybe this should have a TODO / FIXME?